docs/unified-test-documentation/test-matrix.md

```diff
@@ -0,0 +1,148 @@
+# Test matrix
+
+## About
+
+<!--
+Matryca testowa służy do określenia które z zetsawów i przypadków testych
+opisanych w dokumentacji testowej dedykowane są dla danej platformy
+-->
+
+## Test matrix - ASUS KGPE-D16
+
+### Module: dasharo-compatibility
+
+| No.  | Supported test suite                              | Supported test cases                 |
+|:----:|:-------------------------------------------------:|:------------------------------------:|
+| 1.   | coreboot base port                                | All                                  |
+| 2.   | SMBIOS                                            | DMI002.001, DMI003.001               |
+| 3.   | coreboot fan control                              | All                                  |
+| 4.   | custom boot menu key                              | All                                  |
+| 5.   | Debian Stable and Ubuntu LTS support              | LBT001.002                           |
+| 6.   | Network boot                                      | All                                  |
+| 7.   | USB detect                                        | All                                  |
+| 8.   | USB boot                                          | All                                  |
```

Comment:
I understand the concept of tracking information what tests make sense for what platforms, but this format will very quickly become unmaintainable unless there is a way of automatically generate this table for each platform. Add one not compatible test to any of the groups with `All` will make things fall apart, what mechanism we plan to employ to protect against such modification?

Reply:
Also, if new suite will be added how we plan to add it to all machines which are supported?

Reply:
Ultimately, I would like to automate the process of generating such a table for each platform. Automation could be based on a `platform-config` file. In each such file dedicated for a given platform there are flags specifying which tests are executed.